app/components/Grid.tsx

```diff
@@ -119,6 +120,10 @@ export const GridView = ({
     const col = idx % grid.width;
     const row = Math.floor(idx / grid.height);
 
+    const entryCell = entryCells.some((p) => cellIndex(grid, p) === idx);
+    const symmetricalCell = (props.symmetry != Symmetry.None) ? flipped(grid, active, props.symmetry as Symmetry) : null;
```

Comment:
Do we need to check for the case that `props.symmetry` is null/undefined here? I.e. in the case that this grid is being used for a puzzle rather than the builder?

Reply:
I think that would enable us to get rid of the `as` cast, too - which should always be a last resort.

Reply:
shoot I addressed the null case but i had rewritten the line after ... well, college football game 😆
For whatever reason, it does work as-is in a puzzle, but you're right that it should be left in

Reply:
😂